packages/mesos/extra/dcos-mesos-slave.service

```diff
@@ -12,6 +12,7 @@ EnvironmentFile=/opt/mesosphere/environment
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave-common
 EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-common-extras
 EnvironmentFile=/opt/mesosphere/etc/mesos-slave
+EnvironmentFile=-/opt/mesosphere/etc/mesos-slave-extras
```

Comment:
I guess they'd just need different file contents, not different env vars. Ok, I'll remove these two until we have a stronger need for them.